packages/form-handlers-mailchannels-test/src/index.ts

```diff
@@ -13,13 +13,19 @@ export default {
     }
 
     const formData = new FormData();
+    formData.append("ws--form-id", "test-form");
```

Comment:
plz move `ws--form-id` as shared const identifier, examples https://github.com/webstudio-is/webstudio-builder/blob/d61c39f22db07e0171fbc232eddb2d8b650b0dfd/packages/react-sdk/src/tree/webstudio-component.tsx#L61-L63C1